.github/workflows/CI.yml

```diff
@@ -704,6 +704,52 @@ jobs:
           cd testing-framework/terraform
           make terraformCleanup
 
+  run-logs-testbed:
+    runs-on: ubuntu-latest
+    needs: [ e2etest-preparation, create-test-ref]
+
+    steps:
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: '11'
```

Comment:
this should be jdk 17
also can we use `corretto` instead of zulu since we are touching this? 
https://github.com/aws-observability/aws-otel-test-framework/pull/1426/files#diff-a7cefbb47c2967a741e1f19b78a3de4fc74fe9cb6914be4ba8f676e6675150a1R34

Reply:
oh yes good catch !